opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -138,7 +139,7 @@
 
     private FloatingActionButton layerSwitcherFab;
 
-    private LinearLayout progressIndicatorsGroupView;
+    private FloatingActionButton filterTasksFab;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'filterTasksFab' could be replaced by a local variable.](https://app.codacy.com/app/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)